frontend/packages/model-registry/upstream/frontend/src/app/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetails.tsx

```diff
@@ -18,6 +18,8 @@ import ModelVersionSelector from './ModelVersionSelector';
 import ModelVersionDetailsTabs from './ModelVersionDetailsTabs';
 import ModelVersionsDetailsHeaderActions from './ModelVersionDetailsHeaderActions';
 import { MRDeployButton } from '~/odh/components/MRDeployButton';
+import { MRDeploymentsContextProvider } from '~/odh/components/MRDeploymentsContextProvider';
+import { KnownLabels } from '@odh-dashboard/internal/k8sTypes';
```

Comment:
Avoid importing from `@odh-dashboard/internal/k8sTypes` directly because the MR midstream shouldn't know the existence of other packages. If needed, we can recreate this type in `~/odh/k8sTypes`.